Competition/src/main/java/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -178,4 +175,11 @@ public void setUpClimberCommands(OperatorInterface operatorInterface, DeployAllC
         operatorInterface.operatorGamepad.getifAvailable(5).whenPressed(retractBack);
         operatorInterface.operatorGamepad.getifAvailable(6).whenPressed(retractFront);
     }
+
+    @Inject
+    public void setupConfigureDriveSubsystemCommands(ConfigureDriveSubsystemCommand setCurrentLimit,
+            ConfigureDriveSubsystemCommand setVoltageRamp) {
+        setCurrentLimit.includeOnSmartDashboard("Set Current Limit in Amps");
```

Comment:
We only need one command, and add it to the dashboard as "Set Drive Talon Parameters"